python/src/etos_api/library/validator.py

```diff
@@ -191,6 +193,13 @@ async def validate(self, test_suite_url):
                         test_runners.add(constraint.value)
             docker = Docker()
             for test_runner in test_runners:
+                for _ in range(3):
+                    result = await docker.pull(test_runner)
+                    if result:
+                        break
+
```

Comment:
Add a log print maybe? I guess opentelemetry will not work here, but maybe just a print to see when retries are done.

Reply:
Good point, I'll add that.

Reply:
If it's not too hard we should add some data to an otel span here describing how many retries were done.
This is not required. We could add it to metrics later instead.